Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.10-slim
+
+RUN groupadd --gid 1000 appuser \
+    && useradd --uid 1000 --gid 1000 -ms /bin/bash appuser
+
+RUN pip3 install --no-cache-dir --upgrade \
+    pip \
+    virtualenv
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    software-properties-common \
+    git
+
+USER appuser
+ENV VIRTUAL_ENV=/home/appuser/venv
+
+COPY . /home/appuser/document-to-podcast
+WORKDIR /home/appuser/document-to-podcast
+
+RUN virtualenv ${VIRTUAL_ENV}
+RUN . ${VIRTUAL_ENV}/bin/activate && pip install torch torchaudio --index-url https://download.pytorch.org/whl/cpu
```

Comment:
> Is this image meant to be used both by users locally and for our hosted demo?

For now, this is meant to be used for #12. We could later add a base image for users and then build this one from it.

> Maybe for the hosted demo it would make sense to try it on a GPU if we get significant speed improvement.

I think it makes sense for now to "force" ourselves to CPU for the demo. Because of costs but also to showcase the behavior someone could expected on their own laptop.